.github/workflows/main.yml

```diff
@@ -12,22 +12,23 @@ jobs:
   build:
     strategy:
       matrix:
-        jdk: [ 8, 11 ]
+        jdk: [ 8, 11, 17, 21, 22 ]
```

Comment:
```suggestion
        jdk: [ 8, 11, 17 ]
```

Reply:
@Ao-senXiong #15 is basically this. I'll merge this PR and in your PR please look into support for JDK 21.